.circleci/config.yml

```diff
@@ -3,39 +3,28 @@ version: 2.1
 jobs:
     "server-test":
         docker:
-            - image: circleci/python:3.7.9-node-browsers
+            - image: circleci/python:3.9.2-buster-node-browsers
               environment:
                 PERCY_PARALLEL_TOTAL: -1
-            - image: cypress/base:10
-
         steps:
             - checkout
             - run:
                 name: Inject Percy Environment variables
                 command: |
                   echo 'export PERCY_TOKEN="$PERCY_TOKEN_E2E"' >> $BASH_ENV
-
-            - restore_cache:
-                key: dep-{{ .Branch }}-{{ checksum "package-lock.json" }}-{{ checksum "package.json" }}-{{ checksum ".circleci/config.yml" }}
             - run:
                 name: Install npm packages
                 command: npm ci
-            - save_cache:
-                key: dep-{{ .Branch }}-{{ checksum "package-lock.json" }}-{{ checksum "package.json" }}-{{ checksum ".circleci/config.yml" }}
-                paths:
-                    - node_modules
```

Comment:
Is it intentional to drop the cache? Or just temporary while testing?

Reply:
Temporary. Adding back. 🙈